client/src/components/common/Table.tsx

```diff
@@ -14,7 +14,7 @@ export const Table = styled.table`
 
       @media (prefers-color-scheme: dark) {
         background-color: ${(props) =>
-          colorShade(props.theme.colors.backgroundDark, -40)} !important;
+          colorShade(props.theme.colors.background, -40)} !important;
```

Comment:
Hmm, I feel like we should switch away from these variables entirely. Should be a ticket